### PR TITLE
Style updates & DOM changes to reflect them

### PR DESCRIPTION
**Pull request by iisa**
- remove click halos from buttons
- update edit button's size dependency
- Styling & Dom reorg to accommodate updated design
- update demo to reflect bookmark icon standard on archive.org

### CODE REVIEW

File: src/ia-bookmarks-list.js [hunk 1]
```diff
@@ -96,19 +95,38 @@ export class IABookmarksList extends LitElement {
     this.emitDeleteBookmark(id);
   }
 
+  bookmarkColorInfo(colorVal = 0) {
+    return this.bookmarkColors.find((labelInfo) => {
+      console.log('labelInfo', labelInfo);
```
Comment (jbuckner) [inline src/ia-bookmarks-list.js]: There's a console.log in here

File: demo/index.html [hunk 2]
```diff
@@ -72,47 +79,60 @@
...
-      id: 2,
-      thumbnail: '//placehold.it/320x496/06c/fff',
-      page: 9,
       note: 'This is a long comment I left about this bookmark in order to test out the display in the panel on the side of the bookreader.',
-    }, {
-      id: 3,
+      color: 0,
+      id: '2',
+      leafNum: '2',
+      page: '-2-',
```
Comment (jbuckner) [inline demo/index.html:90]: Is this `-2-` a special value?

**Review by jbuckner — APPROVED** [review]
One question, one comment, but looks good overall and QA'd on the review app

File: src/ia-bookmarks-list.js (hunk 1 above)
Comment (iisa) [reply src/ia-bookmarks-list.js]: whoops! thanks, i removed

File: demo/index.html (hunk 2 above)
Comment (iisa) [reply demo/index.html:90]: yes, it is the display for the page.  i am just updating the schema for the demo to reflect what we use in petabox.  Fortunately, we don't have to make the change for this value internally in this repo (bc that is coming next!)

**Pull request merged**